docs/conf.py

```diff
@@ -17,224 +17,226 @@
 # If extensions (or modules to document with autodoc) are in another directory,
```

Comment:
I'm guess all these python changes are black's changes?

Reply:
I'd guess about 90% of the changes to the python files are the formatter.
The last 10% is me fixing minor linting issues that I could be sure wouldn't change the function of the python. Or adding error suppression for lint-errors that I didn't feel comfortable changing on my own.

For example, I removed a lot of unnecessary `else` statements after `return` statements.